source/_integrations/group.markdown

```diff
@@ -58,6 +60,21 @@ Binary sensor, light, and switch groups allow you set the "All entities" option.
 - Otherwise, the group state is `off` if at least one group member is `off`.
 - Otherwise, the group state is `on`.
 
+### Climate groups
+In short, the state shown is using the order in the `HVACMode` enum (except that `off` is the least priority). E.g. when any group member entity is `heat`, the group will also be `heat`. A complete overview of how climate groups behave:
+
+- The group state is `unavailable` if all group members are `unavailable`.
+- Otherwise, the group state is `unknown` if all group members are `unknown` or `unavailable`.
+- Otherwise, the group state is `heat` if at least one group member is `heat`.
+- Otherwise, the group state is `cool` if at least one group member is `cool`.
+- Otherwise, the group state is `heat_cool` if at least one group member is `heat_cool`.
+- Otherwise, the group state is `auto` if at least one group member is `auto`.
+- Otherwise, the group state is `dry` if at least one group member is `dry`.
+- Otherwise, the group state is `fan_only` if at least one group member is `fan_only`.
+- Otherwise, the group state is `off`.
+
+For other modes, the most common mode is reported (e.g. for `PRESET_MODES`, `FAN_MODE` and `SWING_MODE`). The reported temperatures are the averages from the given entities.
```

Comment:
Reported? Not controlled?

Reply:
Also controlled, but then it's the selected mode.
```suggestion
For other modes, the most common mode is reported (e.g. for `PRESET_MODES`, `FAN_MODE` and `SWING_MODE`). The reported temperatures are the averages from the given entities.

When controlling the group, this integration will forward the service call to all entities in the group.
```